web/src/test/java/com/microsoft/applicationinsights/web/extensibility/modules/WebSessionTrackingTelemetryModuleTests.java

```diff
@@ -89,6 +89,7 @@ public static void classCleanup() throws Exception {
 
     // region Tests
 
+    @Ignore
     @Test
```

Comment:
Several things:
 1. Please be consistent with the order of @Ignore and @Test. 
 2. Is it possible to keep these tests alive and to change the generateNewSessions/generateNewUsers programatically? The reason for that is that if one day we'll decide to roll back this value for these members we won't need to recall to un-ignore these tests as well.
 3. Also, if on the other hand we'll decide to drop this code altogether then it would be better to locate all the ignored tests next to each other